network-store-iidm-impl/src/test/java/com/powsybl/network/store/iidm/impl/tck/AreaTest.java

```diff
@@ -14,116 +14,42 @@
 // FIXME remove all the overridden test in this class when Areas are implemented
 public class AreaTest extends AbstractAreaTest {
     @Override
-    public void areaAttributes() {
-        // FIXME
-    }
-
-    @Override
-    public void testSetterGetter() {
-        // FIXME
-    }
-
-    @Override
-    public void testChangesNotification() {
-        // FIXME
-    }
-
-    @Override
-    public void testSetterGetterInMultiVariants() {
-        // FIXME
-    }
-
-    @Override
-    public void testGetAreaBoundary() {
-        // FIXME
-    }
-
-    @Override
-    public void areaInterchangeComputation() {
-        // FIXME
-    }
-
-    @Override
-    public void areaIterableAndStreamGetterCheck() {
-        // FIXME
-    }
-
-    @Override
-    public void addVoltageLevelsToArea() {
-        // FIXME
-    }
-
-    @Override
-    public void addSameVoltageLevelToArea() {
-        // FIXME
-    }
-
-    @Override
-    public void testWithTerminals() {
-        // FIXME
-    }
-
-    @Override
-    public void testAddSameBoundary() {
-        // FIXME
-    }
-
-    @Override
-    public void testWithDc() {
-        // FIXME
+    public void mergeAndFlatten() {
+        // merge is not implemented
     }
 
     @Override
-    public void testRemoveVoltageLevel() {
-        // FIXME
+    public void throwAddVoltageLevelOtherNetwork() {
+        // problem
     }
 
     @Override
-    public void throwAddNewAreaSameType() {
-        // FIXME
+    public void removeArea() {
+        // removed equipments is managed differently in powsybl core implementation
     }
 
     @Override
     public void throwRemovedVoltageLevel() {
-        // FIXME
-    }
-
-    @Override
-    public void throwAddVoltageLevelOtherNetwork() {
-        // FIXME
+        // removed equipments is managed differently in powsybl core implementation
```

Comment:
I think you can make it works. See : 
https://github.com/powsybl/powsybl-network-store/commit/02a3fcb5a31c9cff13d48dfe4b8de738416381c1
And same for other tests